docs/criteria/connectors/rc.md

```diff
@@ -0,0 +1,143 @@
+# Spice.ai OSS Data Connectors - RC Release Criteria
+
+This document defines the set of criteria that is required before a data connector is considered to be of [RC](../definitions.md) quality.
+
+All criteria must be met for the connector to be considered [RC](../definitions.md), with exceptions only permitted in some circumstances (e.g. it would be technically infeasible to add a feature/fix a bug for a particular connector).
+
+## RC Quality Connectors
+
+| Connector | RC Quality | DRI Sign-off |
+| - | - | - |
+| Clickhouse    | ❌ |  |
+| Databricks    | ❌ |  |
+| Delta Lake    | ❌ |  |
+| Dremio        | ❌ |  |
+| File          | ❌ |  |
+| FTP/SFTP      | ❌ |  |
+| GraphQL       | ❌ |  |
+| GitHub        | ❌ |  |
+| HTTP/HTTPS    | ❌ |  |
+| MS SQL        | ❌ |  |
+| MySQL         | ❌ |  |
+| ODBC          | ❌ |  |
+| PostgreSQL    | ❌ |  |
+| Sharepoint    | ❌ |  |
+| Snowflake     | ❌ |  |
+| Spice.AI      | ❌ |  |
+| S3            | ❌ |  |
+| Azure BlobFS  | ❌ |  |
+| Spark         | ❌ |  |
+
+## RC Release Criteria
+
+Some connectors impose different testing requirements, as by their nature they support different feature requirements.
+
+Some connectors are unable to support TPC derived test packages due to their nature (e.g. GitHub Issues).
+These connectors are exempt from running the TPC derived test packages, and rely instead on their general integration test.
+
+This table defines the required features and/or tests for each connector:
+
+| Connector     | [TPC-H Derived Tests (Scale Factor)](#tpc-h) | [TPC-DS Derived Tests (Scale Factor)](#tpc-ds) | [Federation](#federation) | [Data Correctness](#data-correctness) | [Streaming](#streaming) | [Native Schema Inference](#schema-inference) |
```

Comment:
```suggestion
| Connector     | [TPC-H Derived Tests (Scale Factor)](#tpc-h) | [TPC-DS Derived Tests (Scale Factor)](#tpc-ds) | [Federation](#federation) | [TPC Data Correctness](#data-correctness) | [Streaming](#streaming) | [Native Schema Inference](#schema-inference) |
```

Reply:
I would prefer to leave that heading as "Data Correctness". Data Correctness could be expanded to more things than just TPC in the future, and the intention is that although it runs on TPC it tests for data correctness as a whole.

Reply:
Fair point. My idea primarily was to avoid current confusion (i.e. data correctness currently only means TPC data correct).